scripts/playlist-view.lua

```diff
@@ -39,6 +39,7 @@ gallery.config.accurate = false
 opts = {
     thumbs_dir = ON_WINDOWS and "%APPDATA%\\mpv\\gallery-thumbs-dir" or "~/.mpv_thumbs_dir/",
     generate_thumbnails_with_mpv = ON_WINDOWS,
+    mkdir_thumbs = false,
```

Comment:
you can put the default to `true`, same in contact-sheet.lua. 
In any case, I keep the same value in the .lua file and the corresponding .conf file, so that it documents both the available options and their default values.